app/models/publication.rb

```diff
@@ -145,6 +145,12 @@ def self.preferred_versions
                                                  .where(%{NOT EXISTS (SELECT * FROM open_access_locations WHERE open_access_locations.publication_id = publications.id AND open_access_locations.source = '#{Source::SCHOLARSPHERE}')})
                                              }
   scope :activity_insight_oa_publication, -> { oa_publication.with_no_scholarsphere_oa_locations.joins(:activity_insight_oa_files).where.not(activity_insight_oa_files: { location: nil }) }
+  scope :troubleshooting_list, -> {
+    activity_insight_oa_publication
+      .where(%{(open_access_status != 'gold' AND open_access_status != 'hybrid') OR open_access_status IS NULL})
```

Comment:
Just a note that once the stuff that I'm working on is merged in, we'll need to add one more thing here:
```ruby
.where(%{preferred_version != '#{NO_VERSION}' OR preferred_version IS NULL})
```
In my branch, `Publication::NO_VERSION` is defined as `'None'`, which is the value that we set if we find that there are no open access permissions for either the accepted or published version of the publication. This is one more way in which the publication can completely exit the workflow. I'm not sure where/how we want to keep track of this todo item..

Reply:
oops, scratch that ☝️. Publications with a `preferred_version` of `'None'` will technically still be in the workflow until we email the person about it (described in https://github.com/psu-libraries/researcher-metadata/issues/799). So instead of filtering on `preferred_version`, I suppose we'll need to filter on whatever new field we're going to set to indicate the the email has been sent.

Reply:
Will whatever indicates that the email has been sent be part of the activity_insight_oa_publication scope? If so, that will cover it here already. Otherwise, if it's going to be added to steps separately, we can always just make a quick ticket.

Reply:
> Will whatever indicates that the email has been sent be part of the activity_insight_oa_publication scope?

Yeah, I think that it would make sense for it to be part of that scope. I'll just add a note about this to #799.